pkg/gcv/configs/config.go

```diff
@@ -464,17 +467,30 @@ type Configuration struct {
 
 func loadRegoFiles(dir string) ([]string, error) {
 	var libs []string
+	var content []byte
```

Comment:
nit: best practice to declare vars in the minimum scope possible (in this case the for loop).

Reply:
ack